### ISSUE TITLE
Pilon

### ISSUE THREAD

**Issue by nialltho** [ubccr/software-layer#210]
Could you please add Pilon genome assembly polisher? It is the first search result for "pilon"  in GitHub: https://github.com/broadinstitute/pilon

**Comment by tonykew** [comment]
Test build of Pilon version 1.23 complete - more info to follow

**Comment by tonykew** [comment]
Pilon version 1.23 has been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider pilon
----------------------------------------------------------------------------
  pilon: pilon/1.23-Java-11.0.16
----------------------------------------------------------------------------
    Description:
      Pilon is an automated genome assembly improvement and variant
      detection tool

    Properties:
      Bioinformatic libraries/apps

    This module can be loaded directly: module load pilon/1.23-Java-11.0.16
[...]
vortex1$ module load pilon/1.23-Java-11.0.16
To execute Pilon run: java -Xmx8G -jar $EBROOTPILON/pilon.jar
Adjust the memory value according to the size of your input files
vortex1$ 
vortex1$ java -Xmx8G -jar $EBROOTPILON/pilon.jar --help
Pilon version 1.23 Mon Nov 26 16:04:05 2018 -0500

    Usage: pilon --genome genome.fasta [--frags frags.bam] [--jumps jumps.bam] [--unpaired unpaired.bam]
                 [...other options...]
           pilon --help for option details 
[...]
vortex1$ 
```
